Source/SwiftLintBuiltInRules/Rules/Lint/FunctionArgumentsSpacingRule.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftSyntax
+
+@SwiftSyntaxRule
+struct FunctionArgumentsSpacingRule: Rule {
+    var configuration = SeverityConfiguration<Self>(.warning)
+
+    static let description = RuleDescription(
+        identifier: "functions_arguments_spacing",
+        name: "Function Arguments Spacing",
+        description: "Remove the space before the first function argument and after the last argument",
+        kind: .lint
```

Comment:
Please move the test examples into the rule description. They are automatically executed as test and serve documentation purposes.